src/data.js

```diff
@@ -0,0 +1,181 @@
+export const aore = {
```

Comment:
We will get rid of these hardcoded values once we get the logic to use admin level tokens to fetch from the machine accounts or any other sources.